distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -560,6 +560,9 @@ type Decimal
        "round towards positive infinity." If use_bankers=True, then it uses
        "round-half-even", also known as "banker's rounding".
 
+       If decimal_places > 0, `round` returns a Decimal; otherwise, it returns
+       an `Integer`.
```

Comment:
So what happens now with `Number.infinity.round`? I think we should either describe in error conditions that `Arithmetic_Error` is raised or note that here that the type is different in this exception (and `NaN` probably too).

Reply:
Done.